test/chemistry/test_core_hamiltonian_orb_reduce_gsc.py

```diff
@@ -0,0 +1,149 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test Core Hamiltonian Orb Reduce """
+
+import unittest
+
+from test.chemistry import QiskitChemistryTestCase
+from qiskit.aqua.operators import WeightedPauliOperator
+from qiskit.chemistry.drivers import PySCFDriver, UnitsType
+from qiskit.chemistry import QiskitChemistryError
+from qiskit.chemistry.core import TransformationType, QubitMappingType
+from qiskit.chemistry.qubit_transformations import FermionicTransformation
+
+
+class TestCoreHamiltonianOrbReduce(QiskitChemistryTestCase):
```

Comment:
This is no longer core.Hamiltonian test. It should be renamed to `TestFermionicTransformationOrbReduce` I think etc